packaging/debian/rules

```diff
@@ -11,7 +11,7 @@ clean:
 	dh_clean
 	if [ -e build ] ; then rm -rf build ; fi ;
 
-install: build
+install:
```

Comment:
Gotcha, that's good to know. I'm on Rethink's slack, too - didn't know about the #engineering channel though.

Thanks!